roles/addons/prometheus_server/tasks/main.yml

```diff
@@ -6,8 +6,31 @@
     state: present
   with_items:
     - prometheus
+    - grafana
     - alertmanager
     - ipmi_exporter
+    - freeipmi
+    - snmp_exporter
+
+#pour generateur de oid walk pour snmp_exporter     
+- name: Package
+  package:
+    name: "{{ item }}"
+    state: present
+  with_items:
```

Comment:
Please use "loop" 
https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#with-items

Reply:
Do not use loop in package related modules, Ansible recommend passing the list directly in the name

From Ansible documentation https://docs.ansible.com/ansible/latest/modules/yum_module.html :
"When used with a loop: each package will be processed individually, it is much more efficient to pass the list directly to the name option."